uniparc-rest/src/main/resources/uniparc.facet.properties

```diff
@@ -3,6 +3,7 @@ facet.limit=5
 
 facet.uniparc.database.label=Database
 facet.uniparc.database.allowmultipleselection=true
+facet.uniparc.database.limit=-1
 
```

Comment:
It would be better if we could put a description how this limit=-1 works when we already have facet.limit=5

Reply:
If we could the format `f.<fieldname>.facet.limit` then it will help us in making similar change in other config file in future

Reply:
I added it in FacetProperty object. (This is a shared place (common-rest))